src/color.js

```diff
@@ -35,7 +35,7 @@ function unmarshall(color, reference) {
         result.b = rgbArray[2];
         result.a = 1;
     } else {
-        throw new Error(`color format not recognized, color: ${color}`);
+        throw new Error(`tangram-cartocss (unmarshall): color format not recognized, color: ${color}`);
```

Comment:
in order to be consistent with the other errors
```javascript
throw new Error(`Unsupported CartoCSS: : color format not recognized: color: ${color}`);
````

Reply:
No, because, this is not an "unsupported cartocss" error. If this error is thrown it means that:

- The cartoCSS is invalid (it won't work even in raster), 
- Or that there is a bug in tangram.cartocss
- Or that there is a bug in tangram-reference

Reply:
IMO using the error string for this kind of things is a little dangerous!

Reply:
What do you mean?